JobConfig/pionbeam/POT_infinitelifetime.fcl

```diff
@@ -0,0 +1,73 @@
+# Sophie Middleton 2021
```

Comment:
you can replace most of this file with #include POT.fcl, then addind g few lines of modifications. That will be more maintainable.  I also suggest adding 'pion' to the name, maybe POT_infinitepion.fcl

Reply:
Yes, can doe this